python/py-spyder-devel/Portfile

```diff
@@ -2,22 +2,22 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
+PortGroup           python 1.0
 PortGroup           active_variants 1.1
+PortGroup           select 1.0
 
-github.setup        spyder-ide spyder 3.2.8 v
+github.setup        spyder-ide spyder c595543
+version             3.2.8-20180522
 name                py-spyder-devel
-revision            0
 # Preference on mailing list is to use small numbers for epoch.
 # This is already a date code, so sticking with dates.
 epoch               20161005
 
-PortGroup           python 1.0
-
-python.versions     27 35 36
+python.versions     27 34 35 36
```

Comment:
Please do not add old python versions. Let's leave that line the way it was and not add py34.

Reply:
okay, done. I think it would still be good to have some sort of MacPorts policy on which Python versions to support so that this can be done consistently...